maps.go

```diff
@@ -29,6 +29,15 @@ func SortedKeys[K Ordered, T any](m map[K]T) []K {
 	return keys
 }
 
+// MapValueOr returns a value of an entry or a default if
+// not found
+func MapValueOr[K comparable, V any](m map[K]V, key K, def V) (V, bool) {
```

Comment:
I wonder if this should be just `MapValue()` instead

Reply:
I would prefer a `MapValue` over `MapValueOr`

Reply:
> I would prefer a `MapValue` over `MapValueOr`

renamed